### ISSUE TITLE
cron resets twice a day

### ISSUE THREAD

**Issue by stave24** [HabitRPG/habitica#4551]
Hello everyone. So, I'm experiencing an issue with cron - it reset twice during the night resulting to massive hp loss. This happened again last week. Is there something you can do to fix that?

My ID is : c7461f02-d081-4974-97c3-92e2c78af156
##

**Comment by ghost** [comment]
Are you sure you just didn't have two tabs open at the same time? Because if it isn't refreshed over the time of one cron, sometimes the website will appear as though it is giving you hp loss, while it may just be updating.

Also if you are chaining the cron time on the website, there might be some bugs:
http://habitrpg.wikia.com/wiki/Settings#Custom_Day_Start

You should also note that the default cron is taken from the players time zone on their computer.

**Comment by Deyanira41** [comment]
I'm having a problem with cron resetting too early. Last night I checked off several dailies around 22:00 but apparently cron had already turned over b/c my hp dropped and those dailies are already checked off for today. This has happened several times now. I changed my cron time to 2am hoping that it might help. 
Any ideas why this is happening?

my user ID is: 7f11bf45-ae3e-454d-ad03-64bee8b738c2

**Comment by teffff** [comment]
Hi,

I had the same issue. I logged in around 22:15 on my pc (debian w/ iceweasel) yesterday, which immediately triggered cron so I couldn't check off the remaining dailies. Then I logged in with my phone (android app) in the morning and a second cron was triggered, causing lots of damage because obviously none of my dailies was checked. I already had the notion before that cron was sometimes early, but I couldn't pin it down because there's no log. Now I'm in a party so I can track it:

teflon attacks Vice's Shade for 4.3 damage, Vice's Shade attacks party for 3.8 damage. -12 hours ago
teflon attacks Vice's Shade for 0.0 damage, Vice's Shade attacks party for 9.0 damage. -2 hours ago

I don't know where HabitRPG gets my tiime from but the time and zone settings appear to be the same on all my devices. I haven't manually changed when cron was triggered. It appears the problem happened because I logged in between the two trigger times, so maybe I should just set crone to something like 4 in the morning. But I'll wait with that until I've heard what you guys say.

Thanks,
teff
UderID 7577abb7-98c8-4f71-a5d2-1370f5f9b016
